common/transport/amqp/src/amqp.ts

```diff
@@ -132,6 +88,185 @@ export class Amqp {
         forever: false
       }
     }, amqp10.Policy.EventHub));
+
+    this._fsm = new machina.Fsm({
+      initialState: 'disconnected',
+      states: {
+        disconnected: {
```

Comment:
>disconnected [](start = 8, length = 12)

So I note that you are not using a queue for handles like you do in the link code.  ?